public/documents/fari-rpgs/charge-rpg.md

```diff
@@ -2334,6 +2334,300 @@ The original idea comes from [Ironsworn](https://ironswornrpg.com)'s vows. `Goal
+
+#### 3. Allies
+
+This represents people that might be willing to help the party, or who have reason to be connected with the party in some way. While typically positive sounding, negative `dispositions` could be used to represent obligations or relationships turned sour. A positive `key concept` might be something like “friends to all we meet”, while a negative one might be something like “everyone expects our help whether they deserve it or not”. 
+
+**Positive (+) Supporting Details**
+
+Positive `supporting details` for allies could be people or groups that are both friendly to the party and have the means to be helpful. 
+
+**Neutral (0) Supporting Details**
+
+Neutral `supporting details` for allies could be people that are willing to help but don’t have the means or maybe a group that hasn’t decided for sure whether the party is worthy of being helped. 
+
+**Negative (-) Supporting Details**
+
+Negative `supporting details` for allies could be people that you’re connected to that cause more trouble for the party than they might be worth, or estranged relationships. 
+
+#### 4. Adversaries
+
+This represents people or groups that are against the party, or possibly even harsh conditions & impending doom. While typically negative sounding, positive leaning `dispositions` could represent friendly competition or forces of nature that can be treated as both friend and foe. A positive `key concept` might be something like “with friends like these, who needs enemies?”, while a negative one might be something like “hunted by nearly everyone”. 
+
+**Positive (+) Supporting Details**
+
+Positive `supporting details` for adversaries could be people or groups that might still help the party if they were in mortal danger, sworn enemies that are comically bad at their jobs, or maybe enemies that don’t yet realize the party exists.
+
+**Neutral (0) Supporting Details**
+
+Neutral `supporting details` for adversaries could be enemies that have bigger fish to fry and consider the party beneath them, or powerful desert sandstorms that can be used to the party’s advantage if they can survive. 
+
+**Negative (-) Supporting Details**
+
+Negative `supporting details` for adversaries might include powerful enemies sworn to destroy the party, or impending doom of an inbound asteroid.  
+
+## Danger and Progress
+
+> #### Designer Notes
+> 
+> The pace at which the `supporting details` of the party change can be a dial for changing the feel of the game, especially in games where additional mechanical benefits are offered for building up positive `supporting details`.  By default, the intended pace is an average of no more than one detail changing per session.
+
+The `party sheet` holds two clocks which fight against each other, one dragging the party down, and the other pulling the party up. By default, both of these clocks have eight segments. 
```

Comment:
suggestion: OK here's the information I wanted above. I feel like the section above should include this or perhaps both sections should be mixed up somehow ? What do you think?